src/content/changelogs/pages.yaml

```
@@ -5,6 +5,15 @@ productLink: "/pages/"
 productArea: Developer platform
 productAreaLink: /workers/platform/changelog/platform/
 entries:
+  - publish_date: "2024-12-19"
+    title: Cloudflare GitHub App Permissions Update
+    description: |-
+      - Users with the [Cloudflare GitHub App](https://github.com/apps/cloudflare-workers-and-pages) installed to connect repositories to Pages or Workers will be prompted to update permissions **today**. New permissions will allow Cloudflare to automatically create a repository on your GitHub account and deploy the new repository for you. 
```

```suggestion
      - Updated permissions that the [Cloudflare GitHub App](https://github.com/apps/cloudflare-workers-and-pages) requests when installed, allowing you to create new repositories on your GitHub account from the Cloudflare dashboard, when creating new projects. 
       If you have the [Cloudflare GitHub App](https://github.com/apps/cloudflare-workers-and-pages) installed, you will receive an email prompting you to update the permissions you grant the Cloudflare Github App.
```

- 2nd person, not "users"
- "today" is confusing if you imagine reading this changelog entry next week
- Invert it and start with the what and why, then the what will happen
- Can we clarify how this impacts new people?